packages/web/src/scss/components/TextField/_TextField.scss

```diff
@@ -105,7 +90,8 @@
         right: 0;
         bottom: 0;
         left: 0;
-        border: theme.$input-border-width theme.$input-border-style theme.$input-border-color-default;
+        border: form-fields-theme.$box-field-input-border-width form-fields-theme.$box-field-input-border-style
```

Comment:
maybe break lines? So it is more readable?

Reply:
Formatting has another opinion... :/